packages/client/src/publish/ProxyPublisher.ts

```diff
@@ -0,0 +1,30 @@
+import { StreamDefinition } from '../types'
+import { StreamIDBuilder } from '../StreamIDBuilder'
+import { inject, Lifecycle, scoped } from 'tsyringe'
+import BrubeckNode from '../BrubeckNode'
+import { Context } from '../utils/Context'
+import { ProxyDirection } from 'streamr-client-protocol'
+
+@scoped(Lifecycle.ContainerScoped)
+export default class ProxyPublisher {
```

Comment:
We could combine `ProxyPublisher` and `ProxySubscriber` to a single class? Actually the methods in these classes are identical, so it would make it easy to read+maintain if the combined class would just have methods:
```
async setSubscribeProxies(streamDefinition: StreamDefinition, nodeIds: NodeId[], direction: ProxyDirection)
async removeSubscribeProxies(streamDefinition: StreamDefinition, nodeIds: NodeId[], direction: ProxyDirection)
```